tests/astar_test.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+import numpy
+from worldengine import astar
+
+
+class TestCommon(unittest.TestCase):
+
+    def test_traversal(self):
```

Comment:
You're right, there needs to be more documentation in astar.py, if you wish, create an issue otherwise I'll get to in time.

Astar works on cost, the higher the cost the less likely it is to travel that path.  There are no hard limits, it works on minus infinity and positive infinity. It will take a starting position and and end position, then find the path between the two with the lowest cost.  This is perfect for heightmaps for example, because you can use it to find path through mountain/hills between villages. ;)